src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    & {
+      display: grid;
```

Comment:
- You don't need to set grid in media queries as the `.catalog` selector itself has this style
- and it's not necessary to specify `& {`